README.md

```diff
@@ -49,6 +49,42 @@ Run `sh build.sh help` to see all the actions you can perform (build ios/osx, ge
 
 Executing the examples under the `examples/` folder, requires that you have built the `Realm.framework`.
 
+
+## Branches
+
+Here is an overview of some long-running branches we're currently using in development, which you may wish to use.
+
+_:warning: Feedback is very welcome for unreleased version, but please aware that we can't provide support to the same
```

Comment:
Still feels extraneous to me. If you still want to keep it please rewrite the sentence — it’s not proper english at the moment: 

_Warning: we welcome feedback on these branches — please comment on their PRs whenever possible. We strongly recommend against using those branches to ship production applications._